src/game_state/Player.cpp

```diff
@@ -5,7 +5,7 @@
 
 // static
 const float Player::PLATFORM_SPACING = 0.01f;
-const glm::vec3 Player::INITIAL_POSITION(-3, 3, -5);
+const glm::vec3 Player::INITIAL_POSITION(-5.5, 3, -5);
```

Comment:
Makes the player start closer to the left edge of the first platform instead of having to jump immediately after landing. Also makes my game segfault once every 3 resets instead of every other reset